Ports/libtiff/package.sh

```diff
@@ -10,6 +10,7 @@ configopts=(
     "--prefix=/usr/local"
     "--disable-static"
     "--enable-shared"
+    "--disable-webp"
```

Comment:
Does omitting this actually lead to a dependency loop, or do they have some fancy way of resolving the conflict?

Reply:
When I ported libwebp, I started with no support for TIFF files, then I tried to compile it after I successfully ported libwebp without it and `libtiff` refused to install because it found `libwebp` headers.
So it might be that I introduced this loop myself and it can't actually exist in practice here :)
What do you think?

Reply:
Porting over the `libtool` patch allows `libwebp` to be built as a dynamic library, which seems to resolve the dependency problem. As far as I can tell, `libtiff` and `libwebp` can then be built in any order.